Since 0.30.0 sse_mem_clear() switched from SSE SIMD (_mm_storeu_ps) to SSE2 (_mm_storeu_si128)

--- OpenJPH-0.31.0/src/core/codestream/ojph_codeblock_fun.cpp.orig	2026-07-27 09:29:06.000000000 +0200
+++ OpenJPH-0.31.0/src/core/codestream/ojph_codeblock_fun.cpp	2026-09-05 18:08:54.283388739 +0200
@@ -183,8 +183,8 @@ namespace ojph {
     #if (defined(OJPH_ARCH_X86_64) || defined(OJPH_ARCH_I386))
 
       // Accelerated functions for INTEL/AMD CPUs
-      #ifndef OJPH_DISABLE_SSE
-        if (get_cpu_ext_level() >= X86_CPU_EXT_LEVEL_SSE)
+      #ifndef OJPH_DISABLE_SSE2
+        if (get_cpu_ext_level() >= X86_CPU_EXT_LEVEL_SSE2)
           mem_clear = sse_mem_clear;
       #endif // !OJPH_DISABLE_SSE
 
--- OpenJPH-0.31.0/src/core/CMakeLists.txt.orig	2026-09-05 18:11:33.289193997 +0200
+++ OpenJPH-0.31.0/src/core/CMakeLists.txt	2026-09-05 18:11:35.772513877 +0200
@@ -48,13 +48,12 @@ else()
       OR MULTI_GEN_X86_64)
 
       if (NOT OJPH_DISABLE_SSE)
-        list(APPEND SOURCES ${CODESTREAM_SSE} ${TRANSFORM_SSE})
-        source_group("codestream" FILES ${CODESTREAM_SSE})
+        list(APPEND SOURCES ${TRANSFORM_SSE})
         source_group("transform" FILES ${TRANSFORM_SSE})
       endif()
       if (NOT OJPH_DISABLE_SSE2)
-        list(APPEND SOURCES ${CODESTREAM_SSE2} ${TRANSFORM_SSE2})
-        source_group("codestream" FILES ${CODESTREAM_SSE2})
+        list(APPEND SOURCES ${CODESTREAM_SSE} ${CODESTREAM_SSE2} ${TRANSFORM_SSE2})
+        source_group("codestream" FILES ${CODESTREAM_SSE} ${CODESTREAM_SSE2})
         source_group("transform" FILES ${TRANSFORM_SSE2})
       endif()
       if (NOT OJPH_DISABLE_SSSE3)
@@ -91,7 +90,7 @@ else()
         set_source_files_properties(transform/ojph_transform_avx2.cpp PROPERTIES COMPILE_FLAGS "/arch:AVX2")
         set_source_files_properties(transform/ojph_transform_avx512.cpp PROPERTIES COMPILE_FLAGS "/arch:AVX512")
       else()
-        set_source_files_properties(codestream/ojph_codestream_sse.cpp PROPERTIES COMPILE_FLAGS -msse)
+        set_source_files_properties(codestream/ojph_codestream_sse.cpp PROPERTIES COMPILE_FLAGS -msse2)
         set_source_files_properties(codestream/ojph_codestream_sse2.cpp PROPERTIES COMPILE_FLAGS -msse2)
         set_source_files_properties(codestream/ojph_codestream_avx.cpp PROPERTIES COMPILE_FLAGS -mavx)
         set_source_files_properties(codestream/ojph_codestream_avx2.cpp PROPERTIES COMPILE_FLAGS -mavx2)
